src/Integration.Vsix.UnitTests/Analysis/SingleFileMonitorTests.cs

```diff
@@ -415,7 +414,9 @@ public void RealFile_MultipleOperations_NoDuplicates()
                 {
                     File.Delete(filePathToMonitor);
                 }
-                testBody();
+                var numberOfOperations = testBody();
+                numberOfOperations.Should().BeInRange(0, 5);
```

Comment:
As we are tasting for a range on line 427 it could still be possible for some of the iterations to report more than 5 operations and we would not catch it so I added another check here.